modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -152,6 +152,8 @@ PolygonConcentricCircleMeshGeneratorBase::validParams()
                         true,
                         "Volume of concentric circles can be preserved using this function.");
   params.addParam<subdomain_id_type>("block_id_shift", 0, "Integer used to shift block IDs.");
+  params.addParam<bool>(
+      "create_interface_boundaries", true, "Whether the interface boundaries are created.");
```

Comment:
are they interfaces though
seems like they are external boundaries wrt to that mesh?